### PR TITLE
add friendly string support for BSD EX_ exit statusses

### PR DESCRIPTION
**Pull request by poettering**
We make use of these exit codes now, let's add limited pretty printing support for it

### CODE REVIEW

**Comment by poettering** [comment]
```
KVM: entry failed, hardware error 0x0
EAX=00000000 EBX=00000000 ECX=00000000 EDX=000206c1
ESI=00000000 EDI=00000000 EBP=00000000 ESP=00000000
EIP=0000fff0 EFL=00000002 [-------] CPL=0 II=0 A20=1 SMM=0 HLT=0
```

CI failure unrelated

File: TODO [hunk 1]
```diff
@@ -24,6 +24,10 @@ Janitorial Clean-ups:
 
 Features:
 
+* log.c: change handling if an error code == 0 is passed: don't respect pre-set
+  errno in that case, like we currently do, but print error string of
+  "Success".
```
Comment (keszybz) [inline TODO]: I wanted to say "But this is already covered by b29f6480eca0550ba65d30fbece8dd4d4bfe666d", but I see I missed a few spots. I'll submit a PR. I think you can drop this commit though, it should be fixed soon either way.

File: doc/CGROUP_DELEGATION.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
 # Control Group APIs and Delegation
 
+*Intended audience: hackers working on userspace subsystems that require direct
+cgroup access, such as container managers and suchlike.*
```
Comment (keszybz) [inline doc/CGROUP_DELEGATION.md]: such as ... suchlike → such as ... and similar

File: man/systemd.exec.xml [hunk 3]
```diff
@@ -2302,6 +2292,11 @@ StandardInputData=SWNrIHNpdHplIGRhIHVuJyBlc3NlIEtsb3BzLAp1ZmYgZWVtYWwga2xvcHAncy
         </listitem>
       </varlistentry>
     </variablelist>
+
+    <para>For system services, when <varname>PAMName=</varname> is enabled and <command>pam_systemd</command> is part
+    of the selected PAM stack, additional environment variables defined by systemd may be set for
+    services. Specifically, <varname>$XDG_SEAT</varname>, <varname>$XDG_VTNR</varname> may be set, see
```
Comment (keszybz) [inline man/systemd.exec.xml]: <strike>may be set</strike>

**Pull request merged**